reference/global-api-information.md

```diff
@@ -40,3 +42,28 @@ end
 In this example, the given info for version 1.0 would have a `description` of "The first stable version of of this example API.", while the `base_path` would be "/:app_name/v1".
 
 You can use the `base_path` and `base_param` directives to define the base routes and their params for re-use across your whole API, or for a specific version. These are applied "before" any prefixes that you specify in your resources and actions, and will *always* apply, before and independently of any `prefix` that may be defined.
+
+
+## Path-Based Versioning
+
+If you want to version your API using request paths, you can set the `version_using` directive to `:path`, and specify a `base_path` that includes a special `:api_version` placeholder that Praxis will fill in as applicable when configuring the routing for your resources.
```

Comment:
would change to:

If you want to version your API based on request paths, set the `version_using` directive to `:path`, and specify an appropriate `base_path` matcher. This `base_path` matcher must include an `:api_version` variable in it (like any other action route) which Praxis will use to extract the exact version string when routing to your resources.